src/StoreKeeper/WooCommerce/B2C/Frontend/Handlers/Seo/YoastHandler.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace StoreKeeper\WooCommerce\B2C\Frontend\Handlers\Seo;
+
+use StoreKeeper\WooCommerce\B2C\Exceptions\WordpressException;
+use StoreKeeper\WooCommerce\B2C\Frontend\Handlers\Seo;
+use StoreKeeper\WooCommerce\B2C\Options\StoreKeeperOptions;
+use StoreKeeper\WooCommerce\B2C\Tools\WordpressExceptionThrower;
+
+class YoastHandler implements SeoHandlerInterface
+{
+    const YOAST_SEO_OPTION_KEY = 'wpseo_taxonomy_meta';
+
+    public function handle($markdown, $product)
+    {
+        if (!self::isSelectedHandler()) {
+            // revert handler to storekeeper
+            StoreKeeperOptions::set(StoreKeeperOptions::SEO_HANDLER, Seo::STOREKEEPER_HANDLER);
+        }
+    }
+
+    /**
+     * @throws WordpressException
+     */
+    public static function addSeoToPost(int $postId, ?string $title = null, ?string $description = null, ?string $keywords = null): void
```

Comment:
maybe those function should be in the import os some seperate component `src/StoreKeeper/WooCommerce/B2C/Frontend/` namespace is really mend only for front user actions

Reply:
Moved it `Helpers\Seo` only for YoastHandler -> YoastSeo.